Codecs/DAG-PB.md

```diff
@@ -0,0 +1,55 @@
+# [WIP] DagPB Spec
+
+DAG-PB does not support the full ["IPLD Data Model v1."](../IPLD-Data-Model-v1.md)
+
+## Format
+
+The DAG-PB IPLD format is a legacy format implemented with a single protobuf.
+
+```protobuf
+// An IPFS MerkleDAG Link
+message PBLink {
+
+  // multihash of the target object
+  optional bytes Hash = 1;
+
+  // utf string name. should be unique per object
+  optional string Name = 2;
+
+  // cumulative size of target object
+  optional uint64 Tsize = 3;
+}
```

Comment:
Yes. When https://github.com/ipld/js-ipld-dag-pb/pull/127 the JS implementation uses only those names, so that pathing through objects on an IPLD level (not UnixFS/IPFS-level) is the same in Go and JS.